web/themes/joinup/prototype/html-prototype-sandbox/partials/listing-item--event-col-12.mustache

```diff
@@ -1,4 +1,4 @@
-<div class="listing__item listing__item--tile mdl-cell mdl-cell--12-col">
+<div class="listing__item listing__item--tile mdl-cell mdl-cell--12-col mdl-cell--9-col-wide">
```

Comment:
Naming of file vs contents gets tricky `listing-item--event-col-12.mustache`. Not a blocker.